war/src/test/java/io/lumeer/engine/rest/permissions/OrganizationServicePermissionsIntegrationTest.java

```diff
@@ -19,50 +19,322 @@
+      final String code = "TestCreateProjectInOrganizationNoRole_id";
+      final String name = "TestCreateProjectInOrganizationNoRole";
+      organizationFacade.createOrganization(new Organization(code, name));
+
+      String userEmail = userFacade.getUserEmail();
+      String groupName = "TestGroup5";
+      userGroupFacade.addGroups(code, groupName);
+      userGroupFacade.addUser(code, userEmail, groupName);
 
+      securityFacade.removeOrganizationUsersRole(code, Arrays.asList(userEmail), LumeerConst.Security.ROLE_READ);
+      securityFacade.removeOrganizationUsersRole(code, Arrays.asList(userEmail), LumeerConst.Security.ROLE_MANAGE);
+      securityFacade.removeOrganizationUsersRole(code, Arrays.asList(userEmail), LumeerConst.Security.ROLE_WRITE);
+      assertThat(securityFacade.hasOrganizationRole(code, LumeerConst.Security.ROLE_READ)).isFalse();
+      assertThat(securityFacade.hasOrganizationRole(code, LumeerConst.Security.ROLE_MANAGE)).isFalse();
+      assertThat(securityFacade.hasOrganizationRole(code, LumeerConst.Security.ROLE_WRITE)).isFalse();
+
+      final Client client = ClientBuilder.newBuilder().build();
+      Response response = client.target(TARGET_URI)
+                                .path(pathPrefix(code))
+                                .request()
+                                .buildPost(Entity.json(new Project(project, projectName)))
+                                .invoke();
+      String reasonPhrase = response.getStatusInfo().getReasonPhrase();
+      assertThat(reasonPhrase).isEqualTo("Unauthorized");
+      client.close();
    }
 
    @Test
-   public void testCreateProjectInOrganizationWriteRole() {
+   public void testCreateProjectInOrganizationWriteRole() throws UserAlreadyExistsException {
+      final String project = "project2";
+      final String projectName = "Project2Name";
+      final String code = "TestCreateProjectInOrganizationWriteRole_id";
+      final String name = "TestCreateProjectInOrganizationWriteRole";
+      organizationFacade.createOrganization(new Organization(code, name));
+
+      String userEmail = userFacade.getUserEmail();
+      String groupName = "TestGroup6";
+      userGroupFacade.addGroups(code, groupName);
+      userGroupFacade.addUser(code, userEmail, groupName);
 
+      assertThat(securityFacade.hasOrganizationRole(code, LumeerConst.Security.ROLE_WRITE)).isTrue();
+
+      final Client client = ClientBuilder.newBuilder().build();
+      Response response = client.target(TARGET_URI)
+                                .path(pathPrefix(code))
+                                .request()
+                                .buildPost(Entity.json(new Project(project, projectName)))
+                                .invoke();
+      String reasonPhrase = response.getStatusInfo().getReasonPhrase();
+      assertThat(reasonPhrase).isEqualTo("No Content");
+      client.close();
    }
 
+   private String pathPrefix(String organizationCode) {
```

Comment:
I would suggest renaming this to `projectPathPrefix` so the name of the method clearly says what it does.